docs/setup-and-installing.md

```diff
@@ -8,7 +8,7 @@
 
 ## Short version (for SpatialOS local deployment only)
 
-1. Set up your machine by installing: [Unity 2018.2.0b10](https://unity3d.com/unity/beta-download), [Visual Studio 2017](https://www.visualstudio.com/downloads/), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/win), [ReSharper](https://www.jetbrains.com/resharper/) (optional), and [ReSharper Command Line Tools](https://www.jetbrains.com/resharper/download/index.html#section=resharper-clt) (optional).
+1. Set up your machine by installing: [Unity 2018.2.0](https://unity3d.com/get-unity/download/archive), [Visual Studio 2017](https://www.visualstudio.com/downloads/), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/win), [ReSharper](https://www.jetbrains.com/resharper/) (optional), and [ReSharper Command Line Tools](https://www.jetbrains.com/resharper/download/index.html#section=resharper-clt) (optional).
```

Comment:
This change should be done in tandem with an actual upgrade to `2018.2.0` - as we are still using the beta version internally.

Reply:
change the version to `2018.2.0f2`
and the actual upgrade PR went in, so we can now also merge this PR in